climpred/comparisons.py

```diff
@@ -44,6 +44,16 @@ def _drop_members(ds, removed_member=None):
     return ds.sel(member=member_list)
 
 
+def _broadcast_non_CLIMPRED_DIMS_from_forecast_to_verif(forecast, verif):
```

Comment:
needed for new dim `iteration`

Reply:
`xs` doesnt broadcast automatically, only sometimes

Reply:
Do you have an example of this happening? It might be good to add a very small test for this function with a use case.